### ISSUE TITLE
cabal build -c 'tasty-quickcheck < 0.9.1' fails

### ISSUE THREAD

**Issue by Bodigrim** [mitchellwrosen/tasty-hspec#32]
```
$ cabal build -c 'tasty-quickcheck < 0.9.1'
Resolving dependencies...
Warning: Error verifying fetched tarball
/Users/andrew/.cabal/packages/hackage.haskell.org/tasty-quickcheck/0.8.4/tasty-quickcheck-0.8.4.tar.gz,
will redownload:
/Users/andrew/.cabal/packages/hackage.haskell.org/01-index.tar: withFile: does
not exist (No such file or directory)
Downloading  tasty-quickcheck-0.8.4
Downloaded   tasty-quickcheck-0.8.4
Build profile: -w ghc-9.2.5 -O1
In order, the following will be built (use -v for more details):
 - haskell-lexer-1.1.1 (lib) (requires build)
 - tf-random-0.5 (lib) (requires build)
 - tasty-quickcheck-0.8.4 (lib) (requires build)
 - hspec-core-2.10.8 (lib) (requires build)
 - hspec-2.10.8 (lib) (requires build)
 - tasty-hspec-1.2.0.1 (lib) (configuration changed)
Starting     haskell-lexer-1.1.1 (lib)
Starting     tasty-quickcheck-0.8.4 (lib)
Starting     tf-random-0.5 (lib)
Building     tf-random-0.5 (lib)
Building     haskell-lexer-1.1.1 (lib)
Building     tasty-quickcheck-0.8.4 (lib)
Installing   tasty-quickcheck-0.8.4 (lib)
Installing   tf-random-0.5 (lib)
Completed    tasty-quickcheck-0.8.4 (lib)
Completed    tf-random-0.5 (lib)
Installing   haskell-lexer-1.1.1 (lib)
Completed    haskell-lexer-1.1.1 (lib)
Starting     hspec-core-2.10.8 (lib)
Building     hspec-core-2.10.8 (lib)
Installing   hspec-core-2.10.8 (lib)
Completed    hspec-core-2.10.8 (lib)
Starting     hspec-2.10.8 (lib)
Building     hspec-2.10.8 (lib)
Installing   hspec-2.10.8 (lib)
Completed    hspec-2.10.8 (lib)
Configuring library for tasty-hspec-1.2.0.1..
Preprocessing library for tasty-hspec-1.2.0.1..
Building library for tasty-hspec-1.2.0.1..
[1 of 2] Compiling Test.Tasty.Hspec.Compat

src/Test/Tasty/Hspec/Compat.hs:86:50: error:
    Not in scope: ‘T.lookupOption’
    Perhaps you meant ‘Tasty.lookupOption’ (imported from Test.Tasty.Options)
    No module named ‘T’ is imported.
   |
86 |     Tasty.QuickCheck.QuickCheckTests num_tests = T.lookupOption opts
   |                                                  ^^^^^^^^^^^^^^

src/Test/Tasty/Hspec/Compat.hs:87:48: error:
    Not in scope: ‘T.lookupOption’
    Perhaps you meant ‘Tasty.lookupOption’ (imported from Test.Tasty.Options)
    No module named ‘T’ is imported.
   |
87 |     Tasty.QuickCheck.QuickCheckReplay replay = T.lookupOption opts
   |                                                ^^^^^^^^^^^^^^

src/Test/Tasty/Hspec/Compat.hs:88:51: error:
    Not in scope: ‘T.lookupOption’
    Perhaps you meant ‘Tasty.lookupOption’ (imported from Test.Tasty.Options)
    No module named ‘T’ is imported.
   |
88 |     Tasty.QuickCheck.QuickCheckMaxSize max_size = T.lookupOption opts
   |                                                   ^^^^^^^^^^^^^^

src/Test/Tasty/Hspec/Compat.hs:89:53: error:
    Not in scope: ‘T.lookupOption’
    Perhaps you meant ‘Tasty.lookupOption’ (imported from Test.Tasty.Options)
    No module named ‘T’ is imported.
   |
89 |     Tasty.QuickCheck.QuickCheckMaxRatio max_ratio = T.lookupOption opts
   |                                                     ^^^^^^^^^^^^^^
Error: cabal: Failed to build tasty-hspec-1.2.0.1.
```

**Comment by Bodigrim** [comment]
Same for `tasty-hspec-1.2`. Could you please revise lower bounds of `tasty-quickcheck` to be `>= 0.9.1`?

**Comment by mitchellwrosen** [comment]
Sure thing, thanks for the report

**Comment by mitchellwrosen** [comment]
I've revised the bounds of `tasty-hspec-1.2` and `tasty-hspec-1.2.0.1`, and released `tasty-hspec-1.2.0.2` with support for `tasty-quickcheck < 0.9.1` (which probably no one actually needs).

**Comment by mitchellwrosen** [comment]
Please feel free to re-open if I've missed something!
